src/components/Flow.tsx

```diff
@@ -18,10 +18,16 @@ interface IFlowProps {
 export class Flow extends React.PureComponent<IFlowProps> {
   public render(): React.ReactNode {
     return (
+      <div className="Flow">
       <MxGraphContext.Consumer>{(value: IMxGraphContext) => {
         const {
           graph,
         } = value;
+        if (!graph) {
+          // tslint:disable-next-line: no-console
+          console.log("not init graph");
+          return null;
```

Comment:
Do not leak debug statements into production build. In case it is an error scenario, add error handling here(we do not have one, just do "throw"), otherwise you can get rid of it.

I think for this component it is not an error and you can just return null here.